src/pages/Defi/views/StakingVaults.tsx

```diff
@@ -10,9 +11,16 @@ import { EligibleSlider } from '../components/EligibleSlider'
 const DefiHeader = () => {
   const translate = useTranslate()
   return (
-    <Box pb={6}>
-      <Heading>{translate('defi.earn')}</Heading>
-    </Box>
+    <PageHeader>
+      <PageHeader.Left>
+        <Display.Mobile>
+          <PageBackButton />
+        </Display.Mobile>
+      </PageHeader.Left>
+      <PageHeader.Middle>
+        <PageHeader.Title>{translate('defi.earn')}</PageHeader.Title>
+      </PageHeader.Middle>
+    </PageHeader>
```

Comment:
One comment re: boilerplate here, we should probably encapsulate all this into a reusable component since only the title is programmatic

Reply:
Its already a super straight forward composable component. Sometimes it will have a left or right or no middle etc etc.

Reply:
This gives very good readability on what this is doing. And we use it different ways on different pages. It would only apply to a page that has a back button and a heading. But other pages, like LP have a back button, heading, and a right button.